### PR TITLE
Agent version 1.15.2

### PR DESCRIPTION
**Pull request by nmeyerhans**
### Summary

* Bug - Fixed a bug where container state information wasn't reported. [#1076](https://github.com/aws/amazon-ecs-agent/pull/1076)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 #Changelog
 
+## 1.15.2
+* Bug - Fixed a bug where container state information wasn't reported. [#1067](https://github.com/aws/amazon-ecs-agent/pull/1067)
```
Comment (richardpen) [inline CHANGELOG.md]: Can you fix the number, it should be 1076.

Comment (jhaynes) [inline CHANGELOG.md]: This entry should be #1076 rather than #1067.

Comment (nmeyerhans) [reply CHANGELOG.md]: done

Comment (nmeyerhans) [reply CHANGELOG.md]: and done (again)

**Pull request merged**